### PR TITLE
Use a dedicated source.list file to install clever-tools on debian

### PR DESCRIPTION
**Pull request by mbaechler**
It's much more convenient to have a file for each external repository

### CODE REVIEW

**Comment by BlackYoup** [comment]
Hello!

Thanks for this change. Indeed I think that's better too, I'll ask the team about it since we have the same command on our site that would need to be updated (https://www.clever-cloud.com/en/downloads)

**Comment by BlackYoup** [comment]
Could you rename the `clever.list` file to `clevercloud.list`? So that it's aligned with our bintray repository name.

File: commons/clever-tools/getting_started.md [hunk 1]
```diff
@@ -37,14 +37,14 @@ If you are using a GNU/Linux distribution that uses `.deb` packages like Debian
...
-  apt-get install apt-transport-https ca-certificates
+  apt install apt-transport-https ca-certificates
   ```
 
 ```sh
 apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys "379CE192D401AB61"
-echo "deb https://dl.bintray.com/clevercloud/deb stable main" | tee -a /etc/apt/sources.list
-apt-get update
-apt-get install clever-tools
+echo "deb https://dl.bintray.com/clevercloud/deb stable main" | tee -a /etc/apt/sources.list.d/clever.list
```
Comment (mbaechler) [inline commons/clever-tools/getting_started.md:45]:
```suggestion
echo "deb https://dl.bintray.com/clevercloud/deb stable main" | tee -a /etc/apt/sources.list.d/clevercloud.list
```

**Comment by BlackYoup** [comment]
Thanks!

**Pull request merged**